### PR TITLE
Impossible to create goal in empty project

### LINKED ISSUE #992: Impossible to create goal in empty project
image

### PR DESCRIPTION
**Pull request by awinogradov**
## PR includes

- [x] Bug Fix

## Related issues

Resolve #992

## QA Instructions, Screenshots, Recordings
  
- create new project
- try to create new goal in this project
- it works

### CODE REVIEW

**Comment by 9teen90nine** [comment]
мне ок

File: src/utils/db.ts [hunk 1]
```diff
@@ -31,7 +31,7 @@ export const createGoal = async (activityId: string, input: GoalCommon) => {
             ${activityId},
             ${input.state.id},
             ${input.priority},
-            max("scopeId") + 1
+            count(*) + 1
```
Comment (LamaEats) [inline src/utils/db.ts:34]: its counting all rows in goals table, really we're needs this?

Comment (awinogradov) [reply src/utils/db.ts:34]: No all in table, all where projectId === selected project id

Comment (awinogradov) [reply src/utils/db.ts:34]: `FROM "Goal" WHERE "projectId" = ${input.parent.id};`

**Review by LamaEats — APPROVED** [review]

**Pull request merged**